src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedId = (
+    tabs.find(tab => tab.id === selectedTabId) !== undefined
+      ? selectedTabId
+      : tabs[0].id
+  );
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': tab.id === selectedId })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={(tab.id) !== selectedId
```

Comment:
It's a bad practice to use a lot of logic in jsx functions. It's better to create separate function and execute it here